wsmatrix@martin.zurowietz.de/WmOverride.js

```diff
@@ -272,6 +273,11 @@ var WmOverride = class {
       }
    }
 
+   _extendWorkspacesView() {
```

Comment:
I'd rather leave this in `WorkspacesViewOverride.js`. I know it means duplicate code but I think I prefer clear separation between the different override classes. What do you think?

Also this doesn't delete the new functions again on `destroy`.